index.html

```diff
@@ -96,6 +100,145 @@ <h2>Software Developer</h2>
             </a>
         </div>    
         </section>
+<!--Recent work section-->
+        <section class="recent--work--container">
+            <div class="section--tittle">
+                <h2>MY RECENT <br/> WORK</h2>
+                <span class="devider">
+                    <img src="./assets/svg/Devider.svg" alt="Section devider">
+                </span>
+            </div> 
+               
+            <ul class="cards--container">
+
+                <li class="recent--work--card">
+                    <a href="#">
+                        <picture class="card--thumbnail">
+                          <img src="./assets/img/Snapshoot-Portfolio.png" alt="project screenshot">
+                        </picture>
+                    </a>    
+                    <div class="card--content">
+                            <h3>Multi-Post Stories</h3>
+                            <p>
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                            </p>
+                            <div class="tech--container">
```

Comment:
- [ ] **Please use `ul` tag to group all the technologies and `li` tag for each technology**

What this means is that you should use the ul tag to group all the technologies together in one element. Then, inside of that ul element, you should use the li tag to display each technology used.

Why is it important?

Whenever you are trying to group and list things, you should consider using ul and li tags. HTML has the ability to express lists of things, and it helps the Google robot, screen readers, and all manner of users who don't care solely about the presentation of the site understand your content better.